src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-container">
+      <form
+        action="/"
+        method="get"
+        class="search-bar"
+      >
```

Comment:
Keep your attributes correctly formatted. The attributes of the <form> tag should start on new lines with proper 2-space indentation.